jenkins/version-def.sh

```diff
@@ -32,7 +32,7 @@ CUDA_CLASSIFIER=${CUDA_CLASSIFIER:-"cuda11"}
 CLASSIFIER=${CLASSIFIER:-"$CUDA_CLASSIFIER"} # default as CUDA_CLASSIFIER for compatibility
 PROJECT_VER=${PROJECT_VER:-"24.08.0-SNAPSHOT"}
 PROJECT_TEST_VER=${PROJECT_TEST_VER:-"24.08.0-SNAPSHOT"}
-SPARK_VER=${SPARK_VER:-"3.1.1"}
+SPARK_VER=${SPARK_VER:-"3.2.0"}
 SPARK_VER_213=${SPARK_VER_213:-"3.3.0"}
```

Comment:
BTW do you know if anyone relies on the `SPARK_VER_213` somewhere? it seems an unused one

Reply:
Yes seems it's unused